roles/git_ssh_setup/meta/main.yml

```diff
@@ -0,0 +1,56 @@
+galaxy_info:
```

Comment:
Could you change this file so it matches the format in the other roles in this repo

Reply:
Done.